Sources/WhisperKit/Core/WhisperKit.swift

```diff
@@ -211,114 +228,80 @@ open class WhisperKit {
         }
     }
 
-    /// Sets up the model folder either from a local path or by downloading from a repository.
-    public func setupModels(
```

Comment:
Is this deleted for a specific reason? Would be preferred to keep with with some adjustments since it was previously a public function and may be getting called elsewhere.

Reply:
previously `setupModels` would set `modelFolder` property. Right now we have to set either `modelFolder` or / and `mlxModelFolder` depending on the pipeline setup. In this context this function doesn't make sense any more